src/components/divider/index.stories.mdx

```diff
@@ -0,0 +1,45 @@
+import { Meta, Story, Canvas, ArgsTable } from "@storybook/addon-docs";
+
+import Divider from ".";
+
+<Meta title="Components/Divider" component={Divider} />
+
+# Divider
+
+Divider component renders hr HTML tag. Using the hr tag, an HTML page can define a theme break (e.g. a shift of topic).
+The hr element, which is used to divide text or indicate a change in an HTML page, is typically presented as a horizontal rule, [see MDN doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr).
+Divider component also expects _assistiveHidden_ prop that adds adds _aria-hidden_ attribute that can be used to hide non-interactive content from the accessibility API [see MDN doc](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden#:~:text=The%20aria%2Dhidden%20attribute%20can,children%20from%20the%20accessibility%20tree)
```

Comment:
```suggestion
Divider component also expects _assistiveHidden_ prop that adds _aria-hidden_ attribute that can be used to hide non-interactive content from the accessibility API [see MDN doc](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden#:~:text=The%20aria%2Dhidden%20attribute%20can,children%20from%20the%20accessibility%20tree)
```

Reply:
Thanks @JackHowa . Fixed it in [590ecd5](https://github.com/WPMedia/arc-themes-components/pull/145/commits/590ecd591fbb26c7c4872e84f70749a890f941d6)